src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    private static Machine truck = new Truck();
+    private static Machine bulldozer = new Bulldozer();
+    private static Machine excavator = new Excavator();
 
+    public static void main(String[] args) {
+        truck.doWork();
+        excavator.doWork();
+        bulldozer.doWork();
+
+        truck.stopWork();
+        excavator.stopWork();
+        bulldozer.stopWork();
+    }
```

Comment:
```suggestion
    public static void main(String[] args) {
        Machine truck = new Truck();
        Machine bulldozer = new Bulldozer();
        Machine excavator = new Excavator();
        
        truck.doWork();
        excavator.doWork();
        bulldozer.doWork();

        truck.stopWork();
        excavator.stopWork();
        bulldozer.stopWork();
    }
```

let's make them not static